src/index.js

```diff
@@ -181,6 +181,13 @@ promExporterApp.listen(9599, () => {
               throw "Unauthorized";
             }
 
+            // prevent access if no agency is configured on client
+            if (!request.profile?.agency) {
+              log.error("Missing configuration for token client", request.smaug?.app?.clientId);
```

Comment:
Vær opmærksom på hvordan der logges, eller bliver kibana suuur.

andet argument skal være et objekt, så vidt jeg husker..

Overvej at ændre til log.error(`Missing agency in configuration for client ${request.smaug?.app?.clientId}`); eller noget i den stil